src/DDTrace/Http/Urls.php

```diff
@@ -2,12 +2,37 @@
 
 namespace DDTrace\Http;
 
-
 /**
  * A utility class that provides methods to work on urls
  */
 class Urls
 {
+    private static $defaultPatterns = [
+        // UUID's
+        '|\b([0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89AB][0-9a-f]{3}-[0-9a-f]{12})\b|i',
```

Comment:
This is just based on my previous experience, can we make the `-` optional? We slightly increase the risk to get false positives here but it is limited to only hex strings with one exact number of chars, which is perfectly acceptable in my opinion.

I really appreciate the detailed regex considering `[89AB]` from the spec. I would ask you though:
- to add a comment to the [section](https://en.wikipedia.org/wiki/Universally_unique_identifier#Format) in wikipedia explaining it as it surprised me :) and I had to go through a few google link to find it 😄 
- for consistency with the other parts of the expression can we make it lowercase, the pattern in `i`nsensitive so it should make no difference.

Reply:
> This is just based on my previous experience, can we make the - optional?

Good call! There's [a test for this](https://github.com/DataDog/dd-trace-php/pull/442/files#diff-0ebbc6ef112d2cd18640240b6f8557a7R49) and it is obfuscated by the regex that catches "16-512 bit hex hashes". But since we may end up changing/removing that regex based on your other comment above, I'll change the regex to make the dashes optional. :)

> to add a comment to the section in wikipedia explaining it

Good idea! I'll add that. :)

> for consistency with the other parts of the expression can we make it lowercase

Another good idea. I'll change that. :)